spec/controllers/registered_applications_controller_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.describe RegisteredApplicationsController, type: :controller do
+
+  describe "GET#index" do
+    let(:my_user) { create(:user) }
+    before do
+      sign_in(my_user)
+    end
+
+    it "returns http success" do
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+
+    it "assigns the users applications to @applications" do
+      application_1 = create(:application, user: my_user)
+      get :index
+      expect(assigns(:applications)).to eq(my_user.applications)
+    end
+  end
+
+  describe "GET#create" do
```

Comment:
The `create` is being tested with `GET`. What's a more accurate HTTP verb for creating data?